test/dynamo/test_misc.py

```diff
@@ -4721,6 +4722,77 @@ def test_remove_dead_code_with_exn_table_entries(self):
         self.assertEquals(tab[0].end, 4)
         self.assertEquals(tab[0].target, 6)
 
+    @unittest.skipIf(os.name != "posix", "return code check is POSIX only")
```

Comment:
```suggestion
   @cpuOnly
    @unittest.skipIf(os.name != "posix", "return code check is POSIX only")
```

Reply:
Turns out it's enough to use `traceback.format_exc()` to trigger the segfault - no need to spawn a separate process.